integration-tests/src/test/resources/psi-cache/test-processor/src/main/kotlin/TestProcessor.kt

```diff
@@ -0,0 +1,44 @@
+import com.google.devtools.ksp.processing.*
+import com.google.devtools.ksp.symbol.*
+import com.google.devtools.ksp.validate
+import java.io.File
+import java.io.OutputStream
+
+
+class TestProcessor(
+    val codeGenerator: CodeGenerator,
+    val logger: KSPLogger
+) : SymbolProcessor {
+    var rounds = 0
+    override fun process(resolver: Resolver): List<KSAnnotated> {
+        rounds++
+
+        val syms = resolver.getSymbolsWithAnnotation("com.example.Anno").toList()
+
+        if (rounds == 2) {
+            syms.forEach {
+                if (!it.validate()) {
```

Comment:
If we want to reproduce the caching bug, you should be validating at the first round as well.

Reply:
Good catch!